RawSpeed/Common.h

```diff
@@ -133,6 +99,19 @@ inline bool isPowerOfTwo (int val) {
   return (val & (~val+1)) == val;
 }
 
+template<typename T> bool isIn(const T value, const std::initializer_list<T>& list) {
+  for (auto t : list)
+    if (t == value)
+      return true;
+  return false;
+}
+
+// until we allow c++14 code
+template<typename T, typename... Args>
+std::unique_ptr<T> make_unique(Args&&... args) {
```

Comment:
So there is an implementation of C++14 feature in public header, with no guards.
What happens in this headers is included from a source that is compiled with `-std=c++14`+ ?